quanta-t41/config.json

```diff
@@ -35,7 +35,6 @@
     },
     "workflowRoot": "workflows",
     "taskRoot": "tasks",
-    "httpTemplateRoot": "templates",
-    "httpStaticRoot": "static"
```

Comment:
@yyscamper The README says to place the files into static/bios and static/bmc.  

Do you want to create empty folders so someone can upload a skupack that doesn't work, or do you want to remove the static declaration so someone can upload a skupack that doesn't work?  How did you test to confirm your proposed change didn't affect the workflows in the skupack?  Was the extent of your testing limited to POSTing the skupack?